bHY6yh-d46317/cod-pol7d7_bHY6yh-d46317.xml

```diff
@@ -0,0 +1,150 @@
+              <lb ed="#K"/>politica scire opus esse videtur 
+              <lb ed="#K"/>experientia. Experti circa singula 
+              <lb ed="#K"/>iudicant recta opera: et 
+              <lb ed="#K"/>per que et qualiter  perficiuntur intelligunt.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e158">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e158">Idem metaphorum lib. i. ci.</ref> <quote xml:id="bHY6yh-d46317-Qd1e160">Ho<lb ed="#K" break="no"/>minibus 
+              scientia et ars per experien<lb ed="#K" break="no"/>tiam 
+              euenit: experientia enim 
+              <lb ed="#K"/>artem facit: sed inexperientia cau<lb ed="#K" break="no"/>sam: 
+              vt ait polus. Expertos ma<lb ed="#K" break="no"/>gis 
+              proficere videmus sine experien<lb ed="#K" break="no"/>tia 
+              rationem habentibus causa autem 
+              <lb ed="#K"/>est quia experientia singular<lb ed="#K" break="no"/>ium 
+              est cognitio: ars vero vniuersalium.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e177">
+               <cb ed="#K" n="b"/>
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e179">Iesus filius sirach in eccl. c. 34.</ref> 
+              <lb ed="#K"/> <quote xml:id="bHY6yh-d46317-Qd1e181">Qui non est expertus pauca re<lb ed="#K" break="no"/>cognoscit.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e187">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e191">Uirgilius eneid. li. 11. ante medium satura.</ref> <quote xml:id="bHY6yh-d46317-Qd1e191">Experto credite.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e203">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e202">Paulus apostolus epistola. i. ad thessalonicenses
+              <lb ed="#K"/>c. 5. et fi.</ref> <quote xml:id="bHY6yh-d46317-Qd1e207">Omnia probate quod bonum 
+              <lb ed="#K"/>est tenete..</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e214">
+              <lb ed="#K"/> <ref xml:id="bHY6yh-d46317-Rd1e215">Cassiodorus epistolarum: libro. i. e. 23.</ref> <quote xml:id="bHY6yh-d46317-Qd1e218">Bellico homine stirpis est gaudium 
+              <lb ed="#K"/>comprobari. Latet sub ocio lau<lb ed="#K" break="no"/>dabilis 
+              fortitudo: et dum se 
+              <lb ed="#K"/>probandi non habet spacium occulta 
+              <lb ed="#K"/>est lux tota meritorum.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e237">
+              <ref xml:id="bHY6yh-d46317-Rd1e239">Li 4. e.
+              <lb ed="#K"/>48</ref> <quote xml:id="bHY6yh-d46317-Qd1e239">Multo magi opinio est p<lb ed="#K" break="no"/>urgata 
+              quam si desinentibus quere<lb ed="#K" break="no"/>lis 
+              non fuerit impetita.</quote>
+            </p>
+            <p xml:id="bHY6yh-d46317-d1e247">
+              <ref xml:id="bHY6yh-d46317-Rd1e251">Li. 11 e. 2.</ref> 
```

Comment:
Note for future transcriptions. I think we should expand obvious abbreviations. 

So "Li." can be expanded to "Libro" and "e." should be expanded to "epistola"